README.md

```diff
@@ -278,6 +278,33 @@ To install this gem onto your local machine, run `bundle exec rake install`.
 
 ## Releasing
 
+### Github Package Registry
+
+To release a new version follow the steps strictly: 
+
+- Commit all your feature changes
+- Update the version number in `version.rb`,
+- Run bundle install to update the Gemfile.lock
+- Open your PR and get it approved and merged
+- Checkout to main and then run `bin/release`, which will create a git tag for the version, push git commits and tags, and push the `.gem` file to the github package registry.
+
+Before releasing a new version, make sure you have authenticated with the Github package registry. To do so, create a personal access token ([in your Github account settings](https://github.com/settings/tokens))
+
+Then create or add to the existing file ~/.gem/credentials, replacing `TOKEN` with your personal access token.
+
+```
+---
+:github: Bearer TOKEN
+```
+
+Then run 
+
+```sh
+bundle config set --local gem.push_key github
+```
```

Comment:
I'd say this whole section is misleading for this particular gem. Especially since it placed before correct rubygems block. Do we want to keep it here?